src/main/scala/net/snowflake/spark/snowflake/Parameters.scala

```diff
@@ -445,7 +445,13 @@ object Parameters {
       * URL pointing to the snowflake database, simply
       *   host:port
       */
-    def sfURL: String = parameters(PARAM_SF_URL)
+    def sfURL: String = {
+      var url = parameters(PARAM_SF_URL)
+      if (url.matches("https?://.*")) {
+        url = url.substring(url.indexOf("//") + 2)
+      }
+      url
```

Comment:
You can use Regex Group function
https://www.tutorialspoint.com/javaregex/javaregex_capturing_groups.htm

And do you think we can verify the URL format at here?

Reply:
I will update to use Regex Group in PR. It is not necessary to verify the URL format except there is a strong reason.